fiftyone/core/dataset.py

```
@@ -8263,27 +8261,43 @@ def _make_sample(self, d):
         doc = self._sample_dict_to_doc(d)
         return fos.Sample.from_doc(doc, dataset=self)
 
-    def _sample_dict_to_doc(self, d):
+    def _sample_dict_to_doc(self, d, _reload_backing_docs=True):
```

Ruff (0.14.7)

Missing return type annotation for private function `_sample_dict_to_doc`

(ANN202)